app/views/contests/info.html.erb

```diff
@@ -1,20 +0,0 @@
-<% if !@contest.started? && !@contest.start_time.nil? %>
-  <script type="text/javascript">
-  <!--
-    window.setTimeout("window.location.reload(true);", <%= ((@contest.start_time-Time.now())*1000).round %>);
-  //-->
-  </script>
-<% end %>
-
-<% if false && !@contest_relation.nil? %>
-  <h2>Contest Registration</h2>
-
-  <strong>Country: </strong><%= @contest_relation.country_code %>
-  <strong>School: </strong><%= @contest_relation.school.name %>
-<% end %>
-
-<h2>Contest Settings</h2>
-<b>Mode: </b>Closed book (you cannot access other problems on the site while competing)<br>
-<b>Judge feedback: </b>Real-time<br>
-<b>Scoreboard: </b>Live<br>
```

Comment:
I agree that the "info" tab was pretty useless before, but I don't think we should delete it. (I'm fine with redirecting to the "scoreboard" tab by default instead of "info" for non-signed-in users though.)

 - The line "Mode: Closed book" seems useless because the site doesn't support open-book contests, but I think we should keep it so first-time users know they won't have access to their previous submissions. I'm OK with removing this line though.
 - The line "Judging feedback: Real-time" can be removed (judging feedback is always real-time on our site, and also almost everywhere else these days -- the only exception I can think of is Google Code Jam's hidden verdict test cases).
 - "Scoreboard: Live" was previously always the case, but we are adding non-live scoreboards so this line becomes important. Without it, users can't tell whether past contests had live scoreboards or not. Also, it might help users during the contest, if they wonder why there is nobody else on the scoreboard (though they should have already seen the message "The scoreboard is hidden until the end of the contest" before starting). I think we should also add an explanation of hidden scoreboards here, e.g. "Scoreboard: Hidden (you can only see your own score until the end of the contest)".
 - We might want to move some of the information from the rest of the page into the "info" tab, to take up less space on the other tabs. I'm think specifically of the "Owner" line and the finalize/un-finalize buttons, but we might also want to move the start time, end time, and duration (at first I thought they were too important, but then I realised the user probably already saw that information in the contest list).

Reply:
I don't mind restoring the info tab. The owner field can be moved. I feel like finalize should appear on the scoreboard tab (since it basically locks the state of the scoreboard), so I'd prefer to leave it or move it to appear only on the scoreboard tab. I'd would prefer to keep the others visible on all tabs (users could forget or miss it, and they might have been linked directly to a contest page).

Reply:
I restored the info tab and added the explanation of hidden scoreboards.

I suggest that we separately remove the line "Judging Feedback: Real-time", move the "Owner" line into the info tab, and move the finalize/un-finalize buttons into the scoreboard tab.

Reply:
Sure, I'm fine with that.